clang/lib/Basic/Targets/SystemZ.h

```diff
@@ -119,6 +119,12 @@ class LLVM_LIBRARY_VISIBILITY SystemZTargetInfo : public TargetInfo {
                              TargetInfo::ConstraintInfo &info) const override;
 
   std::string convertConstraint(const char *&Constraint) const override {
+    if (llvm::StringRef(Constraint) == "@cc") {
+      auto Len = llvm::StringRef("@cc").size();
```

Comment:
This is a compile-time constant.  Again, in SystemZ.cpp that is hard-coded; why do we need this expression here?